.ci/containers/pre-build-validator/validate.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+pr_number=${PR_NUMBER:-$1}
```

Comment:
`PR_NUMBER:-$1` was only used on the previous PR to account for env only being supplied on old versions of generate diffs. Thus we wanted to wall back to the arg if the env wasn't set. Since we are curating this new it shouldn't be necessary.

`$PR_NUMBER` should suffice

Reply:
Done.